src/Security/Member_Validator.php

```diff
@@ -166,6 +162,7 @@ public function php($data)
                     ),
                     'required'
                 );
+                $valid = false;
```

Comment:
This is a bug fix. We were setting a validation error, but then returning `true` so the member would get updated anyway!
There was no test coverage for this scenario, so I've added some.